packages/@aws-cdk/aws-scheduler-targets-alpha/test/integ.sqs-send-message.ts

```diff
@@ -38,8 +39,8 @@ const message = integ.assertions.awsApiCall('SQS', 'receiveMessage', {
 // Verifies that expected message is received from the queue
 message.assertAtPath(
   'Messages.0.Body',
-  ExpectedResult.exact(payload),
+  ExpectedResult.stringLikeRegexp(payload),
```

Comment:
When I ran this test with `exact` it would fail the assertion. From logs:
```
INFO	Testing equality between "test" and "{\"$Exact\":\"test\"}"

INFO	Assertion Failed: 
{
    "failed": true,
    "assertion": "{\"status\":\"fail\",\"message\":\"!! Expected type object but received string\\n\\\"test\\\"\"}"
}
```

Reply:
From the logs, it sounds like the message in the queue is the text `test`, as expected. I wonder if the assertion logic is correct. Can you verify if the message in the queue is indeed `test` (without quotes)?

Reply:
After looking into this further, I believe `ExpectedResult.exact` is not fully implemented ([code](https://github.com/samson-keung/aws-cdk/blob/dc4bbec03741eea5bb5b69caa22dbaf18f727262/packages/@aws-cdk/integ-tests-alpha/lib/assertions/providers/lambda-handler/assertion.ts#L111-L122)).